spec/controllers/external_protocols_controller_spec.rb

```diff
@@ -78,6 +78,16 @@
       expect(JSON.parse(response.body)['html']).to eq(html_preview)
     end
 
+    it 'return network errors when calling api client' do
```

Comment:
This test is shadowed by the one below, since it's more general error and it's being rescued.